packages/runtime/runtime-utils/src/summarizerNodeContract.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+import {
+    IGraphNode,
+    ISummarizeInternalResult,
+    ISummarizerNode,
+    ISummarizerNodeConfig,
+    ISummarizerNodeWithGC,
+} from "@fluidframework/runtime-definitions";
+import { SummarizerNode } from "./summarizerNode";
+import { ReadAndParseBlob } from "./summarizerNodeUtils";
+import { SummarizerNodeWithGC } from "./summarizerNodeWithGc";
+
+export { ReadAndParseBlob } from "./summarizerNodeUtils";
+
+export interface ISummarizerNodeRootContract {
+    startSummary(referenceSequenceNumber: number, summaryLogger: ITelemetryLogger): void;
+    completeSummary(proposalHandle: string): void;
+    clearSummary(): void;
+    refreshLatestSummary(
+        proposalHandle: string | undefined,
+        getSnapshot: () => Promise<ISnapshotTree>,
+        readAndParseBlob: ReadAndParseBlob,
+        correlatedSummaryLogger: ITelemetryLogger,
+    ): Promise<void>;
+}
+
+export interface IRootSummarizerNode extends ISummarizerNode, ISummarizerNodeRootContract {}
+
+export interface IRootSummarizerNodeWithGC extends ISummarizerNodeWithGC, ISummarizerNodeRootContract {}
```

Comment:
These two will be used outside of "runtime-utils". Should they rather be part of "runtime-definitions"?

Reply:
This is directly related to other comment answer. Seems like we can delay moving it to that package until necessary.